rust/lance-index/src/vector/hnsw/builder.rs

```diff
@@ -510,6 +511,16 @@ struct HNSWBuilderInner {
     visited_generator_queue: Arc<ArrayQueue<VisitedGenerator>>,
 }
 
+impl DeepSizeOf for HNSWBuilderInner {
+    fn deep_size_of_children(&self, context: &mut deepsize::Context) -> usize {
+        self.distance_type.deep_size_of_children(context)
+            + self.params.deep_size_of_children(context)
+            + self.nodes.deep_size_of_children(context)
+            + self.level_count.deep_size_of_children(context)
+        // Skipping the visited_generator_queue
```

Comment:
@BubbleCal do you think `visited_generator_queue` will occupy a lot of space in memory?  I wasn't sure if it was safe to skip that here.